crates/rattler_conda_types/src/package/run_exports.rs

```diff
@@ -0,0 +1,77 @@
+use std::{fs::File, io::Read, path::Path, str::FromStr};
+
+use serde::{Deserialize, Serialize};
+use serde_with::{serde_as, skip_serializing_none};
+
+/// A representation of the `run_exports.json` file found in package archives.
+///
+/// The `run_exports.json` file contains information about the run exports of a package
+#[serde_as]
+#[skip_serializing_none]
+#[derive(Debug, Deserialize, Serialize, Eq, PartialEq, Hash)]
+pub struct RunExports {
```

Comment:
Oh I forgot! Should we call this `RunExportsJson`?

Reply:
Let's do that later for all types? So far this was just moved.